scss/_patterns_buttons.scss

```diff
@@ -119,9 +110,9 @@
       $button-background-color: $color-transparent,
       $button-border-color: $color-transparent,
       $button-text-color: $color-dark,
-      $button-hover-background-color: darken($color-x-light, $hover-background-darken-percentage),
+      $button-hover-background-color: scale-color($color-x-light, $lightness: -$hover-background-darken-amount * 100),
       $button-hover-border-color: $color-transparent,
-      $button-active-background-color: darken($color-light, $active-background-darken-percentage),
+      $button-active-background-color: scale-color($color-x-light, $lightness: -$active-background-darken-amount * 100),
```

Comment:
These are repeated a lot, maybe it would be good to create our own functions for those. So we can have `scale-active-color($color)` and `scale-hover-color($color)` and not repeat lightness value every time.